packages/spectral/src/generators/componentManifest/templates/partials/inputs.ejs

```diff
@@ -0,0 +1,19 @@
+<% inputs.forEach((input) => { -%>
+  <%= input.key %>: {
+    inputType: "<%= input.defaultType %>",
```

Comment:
I'd rename `input.defaultType` to either `input.type` or `input.inputType`. `defaultType` communicates that the type could change.